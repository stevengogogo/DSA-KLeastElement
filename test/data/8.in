340 60
8203 5609 3072 -2381 4762 8501 -9770 5981 7855 -5937 3525 -6277 8410 9107 1374 -8256 6789 9408 84 5235 212 -1336 -4125 -1264 -4195 -1723 7767 -1564 -5223 4541 2287 3448 -531 -7063 -446 1250 3925 8934 4679 -7693 -6690 8510 1702 -3999 1570 7069 -6473 -2574 -7386 6242 1738 89 5001 315 2046 4568 -9445 8698 3436 -5687 -310 1862 1087 1772 5754 2190 9349 -9501 -862 2142 -2730 -3456 6876 -5122 -455 8103 8718 1548 -8105 5474 -8229 -4451 -9246 -7616 -3420 4369 8234 266 3282 -1717 -8277 -8755 -7825 575 5303 -5012 -4708 7194 8319 4324 2394 -897 -1364 -9575 3129 -7113 -3766 -6780 6049 6254 3573 -3535 3619 1354 6951 -2778 5256 -6666 -5517 4162 434 2701 -3350 6414 4100 844 -6329 7793 -9922 -2030 -701 -6140 672 5743 -6399 -6477 -2542 5488 7636 -6764 -442 -7057 -2752 7199 4401 -991 -1487 -329 7731 -555 1801 2770 709 -5645 8825 -6148 9066 -4767 -102 82 9998 -9822 -6750 -868 -6934 221 -1574 -8305 7148 9006 -1015 7050 -133 -1125 -964 8382 -4492 9030 1326 -5 5855 9846 587 8252 -2414 -5613 -1546 5944 -7178 8751 -7850 4056 7129 -8376 -9169 8252 -6078 -136 3401 966 8821 9594 8989 -7435 -9681 -1514 6601 -1946 3612 5012 5981 5157 -8025 -6196 6065 -1037 -5711 -3271 1675 599 -8711 -6436 3500 6124 4378 4645 -2950 -6390 4124 -8257 -6252 1155 -9679 -7404 -111 5184 8221 -7801 -8624 5975 -445 2056 8454 9928 7347 2515 7964 1252 1227 8308 -775 -6412 2574 7375 2904 -7596 3611 4919 1336 3068 -2569 4900 3038 -1770 -587 3313 7611 -6926 -1594 -1849 8753 9515 -3243 1016 -9905 -7307 2815 9774 5959 -6883 3070 9080 5543 -1407 2938 161 3085 9942 3276 -2697 -2754 -1968 -2733 -4482 -9081 -795 6040 3748 -6163 702 -9079 4614 -6456 -7799 3820 -3459 3781 -935 -2573 4526 340 1176 9170 -2833 4501 -7424 8361 -5319 -3808 5362 -2533 -3064 6348 -189 -4857 -1998 -8331 495 6247 -7333 -1475 3008 -2267 6251 -9621 86 -4189 -7643 -8345 -3689
Query 330 334 2
Reverse 222 331
Query 12 243 38
Query 255 273 18
Reverse 88 304
Query 234 234 1
Reverse 29 43
Insert 211 9924
Reverse 281 338
Delete 323
Reverse 304 310
Query 71 184 58
Insert 45 -3847
Delete 341
Insert 167 6580
Query 262 299 18
Insert 117 -6010
Reverse 274 335
Query 107 143 1
Reverse 159 288
Delete 261
Query 5 99 15
Reverse 124 230
Query 300 323 11
Insert 43 -968
Delete 291
Delete 36
Delete 43
Insert 133 1836
Delete 28
Insert 109 -6003
Reverse 38 295
Insert 286 4282
Insert 66 -5516
Reverse 164 242
Delete 94
Reverse 305 324
Insert 89 1881
Delete 2
Query 4 166 77
Reverse 237 303
Reverse 331 338
Insert 268 -3891
Delete 215
Reverse 44 178
Reverse 79 204
Query 145 234 10
Reverse 266 333
Reverse 49 272
Delete 273
Delete 25
Reverse 178 214
Reverse 23 144
Delete 7
Query 252 305 10
Query 248 307 51
Reverse 64 318
Reverse 301 310
Delete 328
Query 139 313 58
