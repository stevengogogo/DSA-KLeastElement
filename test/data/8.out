-2267
-6473
9080
-4767
161
86
-9905
-6412
-111
-446
-7799
-6399
4919
-1717
